controllers/users.js

```diff
@@ -1,6 +1,34 @@
 const chaincodeQuery = require("../models/chaincodes");
 const userQuery = require("../models/users");
 const imageService = require("../services/imageService");
+const fetch = require("node-fetch");
+
+const verifyUser = async (req, res) => {
+  try {
+    const userId = req.userData.id;
+    if (!req.userData?.identityURL) {
+      return res.boom.serverUnavailable("IdentityURL is Missing");
+    }
+    await userQuery.addOrUpdate({ identityStatus: "PENDING" }, userId);
+  } catch (error) {
+    logger.error(`Error while fetching all users: ${error}`);
+    return res.boom.serverUnavailable("Something went wrong please contact admin");
+  }
+  const fetchData = {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      username: req.userData.username,
+    }),
+  };
+  fetch(process.env.IDENTITY_SERVICE_URL, fetchData);
```

Comment:
Also, is there a reason why the already existing [fetch](https://github.com/Real-Dev-Squad/website-backend/blob/develop/utils/fetch.js) util function is not being used instead of adding a new package?

Reply:
Fetch call is outside the try-catch on purpose because we don't care about the response of that call.

Node has no existing fetch